app/assets/stylesheets/common.scss

```diff
@@ -110,13 +110,22 @@ body {
     .#{$namespace}-footer-toolbar {
       padding-top: 0;
       vertical-align: middle;
+      color: $white-color;
     }
 
     .sul-i-download-3 {
       margin-right: 1px;
       padding-right: 1px;
     }
 
+    .#{$namespace}-footer-toolbar .sul-i-download-3 {
```

Comment:
Can these changes be moved to https://github.com/sul-dlss/sul-embed/blob/master/app/assets/stylesheets/file.scss ?